adopt-counter/index.js

```diff
@@ -0,0 +1,66 @@
+const request = require('sync-request')
```

Comment:
Is it me or does this file do two things?  DB connectivity and show UI - Some MVC perhaps?